tests/suite/graceful_recovery_test.go

```diff
@@ -45,7 +47,7 @@ var _ = Describe("Graceful Recovery test", Ordered, Label("functional", "gracefu
 
 	var ngfPodName string
 
-	BeforeAll(func() {
+	BeforeEach(func() {
 		// this test is unique in that it will check the entire log of both ngf and nginx containers
```

Comment:
It looks like this comment may need to be updated now that we are tearing down before each test in the suite